packages/tenant-process/test/testAddCertifiedAttribute.ts

```diff
@@ -0,0 +1,256 @@
+import { fail } from "assert";
+import { attributeKind } from "pagopa-interop-models";
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import { readLastEventByStreamId } from "pagopa-interop-commons-test/index.js";
+import {
+  generateId,
+  Tenant,
+  Attribute,
+  unsafeBrandId,
+  protobufDecoder,
+  TenantCertifiedAttributeAssignedV2,
+  fromTenantKindV2,
+  toTenantV2,
+} from "pagopa-interop-models";
+import { describe, it, expect, vi } from "vitest";
+import {
+  tenantNotFound,
+  attributeNotFound,
+  tenantIsNotACertifier,
+  certifiedAttributeOriginIsNotCompliantWithCertifier,
+  certifiedAttributeAlreadyAssigned,
+} from "../src/model/domain/errors.js";
+import { ApiCertifiedTenantAttributeSeed } from "../src/model/types.js";
+import {
+  currentDate,
+  getMockAuthData,
+  addOneAttribute,
+  addOneTenant,
+  getMockTenant,
+  getMockCertifiedTenantAttribute,
+} from "./utils.js";
+import {
+  postgresDB,
+  attributes,
+  tenants,
+  tenantService,
+} from "./tenant.integration.test.js";
+
+export const testAddCertifiedAttribute = (): ReturnType<typeof describe> =>
+  describe("addCertifiedAttribute", async () => {
+    const tenantAttributeSeed: ApiCertifiedTenantAttributeSeed = {
+      id: generateId(),
+    };
+    const correlationId = generateId();
+    const requesterTenant: Tenant = {
+      ...getMockTenant(),
+      features: [
+        {
+          type: "PersistentCertifier",
+          certifierId: generateId(),
+        },
+      ],
+      updatedAt: currentDate,
+    };
+    const attribute: Attribute = {
+      name: "an Attribute",
+      id: unsafeBrandId(tenantAttributeSeed.id),
+      kind: attributeKind.certified,
+      description: "an attribute",
+      creationTime: new Date(),
+      code: "123456",
+      origin: requesterTenant.features[0].certifierId,
+    };
```

Comment:
```suggestion
    const attribute: Attribute = {
      ...getMockAttribute(),
      id: unsafeBrandId(tenantAttributeSeed.id),
      kind: attributeKind.certified,
      origin: requesterTenant.features[0].certifierId,
    };
```
Use the getMock and only add the properties relevant to the tests